lib/player.js

```diff
@@ -5434,7 +5434,10 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       for (const nativeCue of nativeCues) {
         const cue = shaka.text.Utils.mapNativeCueToShakaCue(nativeCue);
         if (cue) {
-          this.config_.mediaSource.modifyCueCallback(cue, null, time);
+          const modifyCueCallback = this.config_.mediaSource.modifyCueCallback;
```

Comment:
Please add comments referencing this PR/issue, so that nobody unknowingly remove the workaround.  It might also be good to reference TypeScript tracking issue #8262 in the comment, so we can remove this workaround during the TypeScript migration.  Thanks!